src/SFA.DAS.ApprenticePortal/Models/RolesAndResponsibilitiesConfirmations.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticePortal.Models
+{
+    [Flags]
+    public enum RolesAndResponsibilitiesConfirmations : byte
+    {
+        ApprenticeRolesAndResponsibilitiesConfirmed = 1,
+        EmployerRolesAndResponsibilitiesConfirmed = 2,
+        ProviderRolesAndResponsibilitiesConfirmed = 4
+    }
+}
```

Comment:
think it can be removed

Reply:
thought id already removed it - removed now